### DISCUSSION TITLE
希望增加配置备份的选项

### DISCUSSION THREAD

**Discussion by louiejordan** [Kuingsmile/PicHoro#69] [Ideas]
图床多了还是备份一下省心

**Comment by Kuingsmile** [comment]
![image](https://github.com/Kuingsmile/PicHoro/assets/96409857/b8fe9036-c066-4eec-8bd7-1c3e1f389a7e)
可以在图床设置界面里按右下角按钮全部导出配置到剪贴板，然后自己备份到文件里，不过少了个导入功能，后面我会加上
